index.ios.js

```diff
@@ -27,9 +27,10 @@ class findAR extends Component {
   render() {
     const menu = <Menu onItemSelected={this.onMenuItemSelected} navigator={navigator}/>;
     return (
-      <Provider store= {store}>
+      <Provider store={store}>
         <SideMenu menu={menu}>
-          <ViewContainer/>
+          <ViewContainer>
```

Comment:
just side note: this will later be switched over to render the smart container instead. Since smart container isn't finished, this is ok for now/testing